SimPEG/Props.py

```diff
@@ -307,14 +307,14 @@ def get_property(self):
 
         def fget(self):
             if scope.physical_property is None:
-                return Utils.Zero()
+                return None
```

Comment:
we return None's for the Maps if they don't exist, so doing the same for the derivs would seem to make sense - see what you think @rowanc1